package.json

```diff
@@ -149,14 +98,17 @@
     "vscode-builtin-theme-red": "https://open-vsx.org/api/vscode/theme-red/1.62.3/file/vscode.theme-red-1.62.3.vsix",
     "vscode-builtin-theme-solarized-dark": "https://open-vsx.org/api/vscode/theme-solarized-dark/1.62.3/file/vscode.theme-solarized-dark-1.62.3.vsix",
     "vscode-builtin-theme-tomorrow-night-blue": "https://open-vsx.org/api/vscode/theme-tomorrow-night-blue/1.62.3/file/vscode.theme-tomorrow-night-blue-1.62.3.vsix",
-    "vscode-builtin-typescript": "https://open-vsx.org/api/vscode/typescript/1.45.1/file/vscode.typescript-1.45.1.vsix",
-    "vscode-builtin-typescript-language-features": "https://open-vsx.org/api/vscode/typescript-language-features/1.45.1/file/vscode.typescript-language-features-1.45.1.vsix",
-    "vscode-builtin-vb": "https://open-vsx.org/api/vscode/vb/1.45.1/file/vscode.vb-1.45.1.vsix",
-    "vscode-builtin-icon-theme-seti": "https://open-vsx.org/api/vscode/vscode-theme-seti/1.45.1/file/vscode.vscode-theme-seti-1.45.1.vsix",
-    "vscode-builtin-xml": "https://open-vsx.org/api/vscode/xml/1.45.1/file/vscode.xml-1.45.1.vsix",
-    "vscode-builtin-yaml": "https://open-vsx.org/api/vscode/yaml/1.45.1/file/vscode.yaml-1.45.1.vsix",
+    "vscode-builtin-extensions-pack": "https://open-vsx.org/api/eclipse-theia/builtin-extension-pack/1.50.1/file/eclipse-theia.builtin-extension-pack-1.50.1.vsix",
     "vscode-editorconfig": "https://open-vsx.org/api/EditorConfig/EditorConfig/0.14.4/file/EditorConfig.EditorConfig-0.14.4.vsix",
     "vscode-eslint": "https://open-vsx.org/api/dbaeumer/vscode-eslint/2.1.1/file/dbaeumer.vscode-eslint-2.1.1.vsix",
     "vscode-references-view": "https://open-vsx.org/api/ms-vscode/references-view/0.0.47/file/ms-vscode.references-view-0.0.47.vsix"
```

Comment:
I believe we can update this extension to around (at least) 0.0.81. We have [fixed a bug](https://github.com/eclipse-theia/theia/pull/10491) that prevented the icon from showing correctly, and otherwise I have tested it fairly extensively.

Reply:
I updated to the latest `v0.0.89` and confirmed it works 👍